extent.go

```
@@ -108,6 +109,19 @@ func (en *ExtentNavigator) Read(offset uint64) (data []byte, err error) {
 		}
 	}()
 
+	// If the inode is not using extents, its data is stored in inode.i_block.
+	if !en.inode.Flag(InodeFlagExtents) {
+		if en.inode.Size() > uint64(len(en.inode.Data().IBlock)) {
+			log.Panicf("Inode size is %v bytes but does not use extents", en.inode.Size())
```

This method returns an error. Is there a reason to panic here instead of returning a new error?

---

Just adhering to the panic recover model this library adopts. (Did it for consistency)
The library panics in case of error, the deferred function on the top of each function catches it and returns the error.

---

Getting rid of the panic recover model can be another PR later.